home/tests/import-export-data/broken_assessment.csv

```diff
@@ -1,2 +1,2 @@
 this,is,not,a,valid,content,csv
-"For real, it's totally not."
+"For real, it's totally not.",,,,,,
```

Comment:
The point of this file is to be a broken csv -- adding the commas makes it not broken anymore. 😅